assets/dev/js/editor/controls/repeater-row.js

```diff
@@ -45,6 +45,18 @@ module.exports = Marionette.CompositeView.extend( {
 		return elementor.getControlView( controlType );
 	},
 
+	getChildControlView( name ) {
+		return this.getControlViewByModel( this.getControlModel( name ) );
+	},
+
+	getControlViewByModel( model ) {
+		return this.children.findByModelCid( model.cid );
+	},
```

Comment:
```suggestion
	getControlViewByModel( modelId ) {
		return this.children.findByModelCid( modelId );
	},
```

Will be better to not pass all the model if not need

Reply:
I did this to be consistent with the controls-stack.js, would u prefer to change it everywhere? or not be consistent with this?